concordium-android-sdk-examples/app/src/main/java/com/example/android_sdk_example/Storage.kt

```diff
@@ -0,0 +1,46 @@
+package com.example.android_sdk_example
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.activity.ComponentActivity
+import cash.z.ecc.android.bip39.Mnemonics
+import cash.z.ecc.android.bip39.toSeed
+import com.concordium.sdk.crypto.wallet.ConcordiumHdWallet
+
+/***
+ * Class to hide storage details from the rest of the project.
```

Comment:
Please update the documentation to also state that this is not a safe way of storing a seed phrase, and it is just done for demonstration purposes.